README.md

```diff
@@ -5,38 +5,58 @@
 </p>
 
-[![Join Slack](https://img.shields.io/badge/SkyPilot-Join%20Slack-blue?logo=slack)](https://join.slack.com/t/skypilot-org/shared_invite/zt-1i4pa7lyc-g6Lo4_rqqCFWOSXdvwTs3Q)
-![pytest](https://github.com/skypilot-org/skypilot/actions/workflows/pytest.yml/badge.svg)
-[![Documentation Status](https://readthedocs.org/projects/skypilot/badge/?version=latest)](https://skypilot.readthedocs.io/en/latest/?badge=latest)
+<p align="center">
+  <a href="https://skypilot.readthedocs.io/en/latest/"> 
+    <img alt="Documentation" src="https://readthedocs.org/projects/skypilot/badge/?version=latest">
+  </a>
+  
+  <a href="https://github.com/skypilot-org/skypilot/releases"> 
+    <img alt="GitHub Release" src="https://img.shields.io/github/release/skypilot-org/skypilot.svg">
+  </a>
+  
+  <a href="https://join.slack.com/t/skypilot-org/shared_invite/zt-1i4pa7lyc-g6Lo4_rqqCFWOSXdvwTs3Q"> 
+    <img alt="Join Slack" src="https://img.shields.io/badge/SkyPilot-Join%20Slack-blue?logo=slack">
+  </a>
+  
+</p>
+
+
+<h3 align="center">
+    Run jobs on any cloud, easily and cost effectively
+</h3>
 
 SkyPilot is a framework for easily running machine learning workloads[^1] on any cloud. 
 
-Use the clouds **easily** and **cost effectively**, without needing cloud infra expertise.
+SkyPilot abstracts away cloud infra burden:
+- Launch jobs & clusters on any cloud (AWS, Azure, GCP)
+- Find availability across zones/regions/clouds
+- Queue jobs & use cloud object stores
 
-_Ease of use_
-* **Run existing projects on the cloud** with zero code changes
-* Use a **unified interface** to run on any cloud, without vendor lock-in (currently AWS, Azure, GCP)
-* **Queue jobs** on one or multiple clusters
-* **Automatic failover** to find scarce resources (GPUs) across regions and clouds
-* **Use datasets on the cloud** like you would on a local file system 
+SkyPilot cuts your cloud costs:
+* [Managed Spot](https://skypilot.readthedocs.io/en/latest/examples/spot-jobs.html): **3x cost savings** using spot VMs, with auto-recovery from preemptions
+* [Autostop](https://skypilot.readthedocs.io/en/latest/reference/auto-stop.html): hands-free cleanup of idle clusters 
+* [Benchmark](https://skypilot.readthedocs.io/en/latest/reference/benchmark/index.html): find best VM types for your jobs
+* Optimizer: **2x cost savings** by auto-picking best prices across zones/regions/clouds
 
-_Cost saving_
-* Run jobs on **spot instances** with **automatic recovery** from preemptions
-* Hands-free cluster management: **automatically stopping idle clusters**
-* One-click use of **TPUs**, for high-performance, cost-effective training
-* Automatically benchmark and find the cheapest hardware for your job
+SkyPilot supports your existing GPU, TPU, and CPU workloads, with no code changes. 
+
+Install with pip (choose your clouds)
+```
+pip install "skypilot[aws,gcp,azure]"
+```
+or [install from source](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html).
```

Comment:
This hanging line does not look good to me. Can we merge it with the L43?

Reply:
Done.